jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/PushBuilderImpl.java

```diff
@@ -23,6 +23,7 @@
 import org.eclipse.jetty.http.MetaData;
 import org.eclipse.jetty.util.URIUtil;
 
+@Deprecated(forRemoval = true, since = "12.1.0")
```

Comment:
Ditto remove?

Reply:
OK I'll remove here. I'll just directly commit to the branch as we are doing with other 6.1 changes.  I am only PRing this change because it affects core.